### ISSUE TITLE
taco 성공 후 wavenet 실패

### ISSUE THREAD

**Issue by ocean-park** [hccho2/Tacotron-Wavenet-Vocoder#9]
안녕하세요
소스 잘 사용하고 있습니다. tacotron2 (한개 보이스 학습하여) 는 잘 동작합니다. 합성도 어느정도 되는 것 같구요
그런데 wavenet 학습 진행을 못하겠는데요
약간 설명 드리자면

./data/gosData 폴더에 npz 이 들어있습니다

한 개의 보이스만 훈련하기 위해서 
train_vocoder.py 에
DATA_DIRECTORY =  '.\\data\\gosData'
hparams.py
model_type = 'single',

으로 변경 후 수행 했습니다만
train_vocoder.py 내에서
audio_batch, lc_batch = reader.inputs_wav, **self.local_condition**
self.local_condition을 찾을 수 없다고 출력되고 종료 됩니다.
train_vocoder.py 안에 변수로 선언돼 있지 않아서 어떤 값으로 대제 해야할지 모르겠습니다.

만일 이게 복잡하다면, 제 데이터인  gosData 에 moon 데이터를 같이 학습한 것을 single tacotron 학습한 mel 로 사용해도 잘 동작 할까요?

**Comment by hccho2** [comment]
mel-spectrogram이 local condition으로 입력되어야 합니다.

train_vocoder.py에 있는 DATA_DIRECTORY가  잘 설정되어야 합니다.

**Comment by ocean-park** [comment]
네 잡음이 있지만 wavenet도 동작하는 것 샅아요. 다만 잡음 때문에 원본 데이터를 좀더 정제해서 해보려 하고 있습니다. 소스에는 24000으로 돼있는 걸 44100으로 학습하는데, 소스도 모노로 바꾸고, 여러가지 시도를 하고 있습니다.
